client/src/command/health.rs

```diff
@@ -0,0 +1,45 @@
+use crate::config::{DEFAULT_NODE_ADDR, DEFAULT_PROTOCOL};
+use crate::node::{HealthResponse, DEFAULT_HEALTH_ENDPOINT};
+use clap::ArgMatches;
+use lazy_static::lazy_static;
+
+lazy_static! {
+    static ref DEFAULT_URL: String = format!(
+        "{}://{}/{}",
+        DEFAULT_PROTOCOL, DEFAULT_NODE_ADDR, DEFAULT_HEALTH_ENDPOINT
+    );
+}
+
+pub(crate) const HEALTH_CHECK_ENDPOINT_KEY: &str = "NODE_ENDPOINT";
```

Comment:
Let's build a trait for commands where you define a key, a set of expected args, the request and response, and a handler so we can implement them in a general way. Also we may need to support a chain of subcommands (to make calls such as `./ipc-client checkpoint list`